### PR TITLE
Release 2.10 #167 update chart/docker image versions & helm dependencies

### PR DESCRIPTION
**Pull request by planetf1**
Signed-off-by: Nigel Jones <nigel.l.jones+git@gmail.com>

- egeria static ui 2.8.0 (latest available) (compose / lab chart)
- egeria react ui 2.10.0 (compose/2 helm charts)
- updates bitnami kafka to 12.18.3
- optimizes container loading for stable release (IfNotPresent)
- Addresses #5241 (not working yet --- ....)

### CODE REVIEW

File: open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/files/egeria-ui/application.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project.
+title = 'Open Metadata Platform:|Find the right data with governance. (dev)'
+description = 'Have a question?||Visit https://github.com/odpi/egeria .@@What is Open Metadata?||Start your customer journey with this link to know what is Open Metadata and how can you use this applicaiton.@@Have more cool ideas?||Feel free to let us know your ideas and business challenges so we can make it better.'
```
Comment (lpalashevski) [inline open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/files/egeria-ui/application.properties:4]: Let's revert this change with the deployment and bring the fix from master, please charry-pick: https://github.com/odpi/egeria/pull/5245/commits/1b7b925dbf4f870595475f62ba773ff75efeaf4b

**Review by lpalashevski — CHANGES_REQUESTED** [review]

File: open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/files/egeria-ui/application.properties (hunk 1 above)
Comment (planetf1) [reply open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/files/egeria-ui/application.properties:4]: Yes agreed. Will do. Thanks for proposing the fix

**Comment by planetf1** [comment]
The RAT error is not occuring on a local MacOS build. retrying actions

**Review by lpalashevski — APPROVED** [review]
